src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -37,6 +37,8 @@
 
 logger = get_logger(__name__)
 
+FILE = ""
```

Comment:
Dunno what the CLI team's standard is for this but I wouldn't use globals except for constants, this should just be a separate function to return the file name and pass it around.

Reply:
Right. The FILE name is actually consisted of a combination of library functions including os.times() which captures the current time in seconds. But each time you call os.times(), a different number gets generated. And giving that the FILE is shared in two different functions, I used a global variable FILE.